data/sheragi/sheragi outfits.txt

```diff
@@ -150,7 +150,14 @@ outfit "Electronic Warfare System"
 	"outfit space" -6
 	"unique" 1
 	"radar jamming" 10
-	"tactical scan power" 25
+	"crew scan power" 20
+	"energy scan power" 40
+	"fuel scan power" 30
+	"maneuver scan power" 25
+	"range finder power" 1
+	"strategic scan power" 20
```

Comment:
```suggestion
```

Just a note here: "Strategic Scan" is, like "Tactical Scan" actually a conglomerate of the various movement related information types. So having it here alongside them doubles up on things.